docs/example.rst

```diff
@@ -54,4 +68,7 @@ The code, which is available as a Python file here_, will create this plot:
 
     Environmental contour with a return period of 50 years.
 
-.. _here: https://github.com/virocon-organization/viroconcom/blob/master/examples/sea_state_iform_contour.py
+.. _examples: https://github.com/virocon-organization/virocon/tree/master/examples
+.. _here: https://github.com/virocon-organization/virocon/blob/master/examples/hstz_contour_simple.py
+.. [1] •	Haselsteiner et. Al. (2020): Haselsteiner, A.F.; Sander, A.; Ohlendorf, J.H.; Thoben, K.D. (2020). Global hierarchical models for wind and wave contours: physical interpretations of the dependence functions. OMAE 2020, Fort Lauderdale, USA. Proceedings of the 39th International Conference on Ocean, Offshore and Arctic Engineering.

```

Comment:
Reference [1] does not show up correctly in the HTML.

For some reason it look like "1(1,2) Haslesteiner . ..."

Reply:
If you cite literature twice in doctoring, apparently it looks like this: 

<img width="782" alt="Bildschirmfoto 2021-07-12 um 09 28 59" src="https://user-images.githubusercontent.com/57865209/125247847-af3ef080-e2f3-11eb-8ffe-2226f853ffd7.png">

Source: https://www.python.org/dev/peps/pep-0257/#references-and-footnotes

Reply:
If you cite the same literature twice in docstrings, apparently it looks like this: 

<img width="782" alt="Bildschirmfoto 2021-07-12 um 09 28 59" src="https://user-images.githubusercontent.com/57865209/125248073-edd4ab00-e2f3-11eb-833c-add1d8d930e7.png">


Source: https://www.python.org/dev/peps/pep-0257/#references-and-footnotes